palisade-service/pom.xml

```diff
@@ -32,6 +32,9 @@
     <properties>
```

Comment:
There a number of features in this Maven pom file that need to be changed and be consistent with the other maven files.  Review and make changes based on what has been done in the other maven files.